src/number/NumberFormatter/index.ts

```diff
@@ -0,0 +1,369 @@
+ *  <li> <span style="font-weight: bold;">formatRupeeToPaisa</span> => Converting number from rupee to paisa </li>
+ *  <li> <span style="font-weight: bold;">plainNumber</span> => Converting only to number </li>
+ * </ul>
+ * <br/>
+ * <div>
+ * <div style="font-weight:bold">defaultNumberConfig = { <br/></div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">addCommas: </div>
+ *    <div style="margin-left:4px">true</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">millionCommas: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">fallback: </div>
+ *    <div style="margin-left:4px">'NA'</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">toFixedValue: </div>
+ *    <div style="margin-left:4px">2</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">formatToLakhCrore: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">formatToBillionTrillion: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">decimals: </div>
+ *    <div style="margin-left:4px">2</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">isCurrency: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">currency: </div>
+ *    <div style="margin-left:4px">'INR'</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">absoluteValue: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">roundValue: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">withSign: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">spaceBetweenSignValue: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">formatPaisaToRupee: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">formatRupeeToPaisa: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * <div style="display:flex;">
+ *   <div style="font-weight:bold">plainNumber: </div>
+ *    <div style="margin-left:4px">false</div>
+ * </div>
+ * };
+</div>
+ *
+ * @example
+ * ```
+ * NumberFormatter(100023); // '1,00,023.00'
+ * NumberFormatter(100023,PRICE_CURRENCY);  // '₹1,00,023.00'
+ * NumberFormatter(100023,PRICE_CURRENCY_TO_FIXED_ZERO); // '₹1,00,023'
```

Comment:
toFixed will round off also. That is ... 1.49.toFixed(1) = 1.5

Do we have something that does not round off .... 1.49.toFixedNoRound(1) = 1.4

???

Reply:
toFixedWithoutRounding we have in this function @saxenapeeyush you can use this